app/jobs/exception_hunter/send_slack_notification_job.rb

```diff
@@ -0,0 +1,31 @@
+module ExceptionHunter
+  class SendSlackNotificationJob < ApplicationJob
```

Comment:
I'd do a small test for this, just to ensure the right attributes are passed to the `notifier.ping`